pkg/crc/preflight/preflight_checks_darwin.go

```diff
@@ -319,5 +319,5 @@ func fixPlistFileExists(agentConfig launchd.AgentConfig) error {
 		logging.Debugf("failed to restart launchd agent '%s': %v", agentConfig.Label, err.Error())
 		return err
 	}
-	return nil
+	return checkIfDaemonIsServing()
```

Comment:
This function `checkIfDaemonIsServing()` should be called from `fixDaemonPlistFileExists` instead of `fixPlistFileExists` 

its more related to the daemon plist file then a generic plist file